packages/core/upload/admin/src/utils/getBreadcrumbDataCM.ts

```diff
@@ -0,0 +1,53 @@
+import { getTrad } from './getTrad';
+
+import type { Folder } from '../../../shared/contracts/folders';
+import type { MessageDescriptor } from 'react-intl';
+
+interface BreadcrumbDataFolder extends Omit<Folder, 'children' | 'files' | 'parent'> {
+  parent?: BreadcrumbDataFolder;
+  children?: {
+    count: number;
+  };
+  files?: {
+    count: number;
+  };
+}
+
```

Comment:
Compared to the original Folder type, is the difference just that these are optional now? If so I would recommend extending the type with Partial instead

Reply:
nope, the entire structure is different because in the original Folder is
parent?: number | null | Folder; 
instead children is
children?: number[];
and files is
files?: File[];